rust/sealevel/environments/local-e2e/sealeveltest1/core/program-ids.json

```diff
@@ -3,6 +3,7 @@
   "validator_announce": "DH43ae1LwemXAboWwSh8zc9pG8j72gKUEXNi57w8fEnn",
   "multisig_ism_message_id": "2YjtZDiUoptoSsA5eVrDCcX6wxNK6YoEVW7y82x5Z2fw",
   "igp_program_id": "GwHaw8ewMyzZn9vvrZEnTEAAYpLdkGYs195XWcLDCN4U",
+  "igp_program_data": "HBB1MxTeuRGb3C4bfXLQbu9z2Pn2LTXVQX53Hy7uyeof",
```

Comment:
were these manually added? if so, they'll be overwritten by the CLI - this file corresponds to https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/f1c1910a77c70ec424655f799ea9475610e8432d/rust/sealevel/client/src/core.rs#L360-L373

If this isn't needed I'd recommend just removing it as the igp_program_data can be found like how we do here https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/2585/files#diff-4da632cca4500e4ca83baf054012f69be1d1298986be83c61211884067a28173R50